### PR TITLE
 create a noyan.txt file

### PR DESCRIPTION
**Pull request by noyon57**

### CODE REVIEW

**Review by Neaz-Mahmood — APPROVED** [review]
1st letter of the name should be capital letter

File: noyan.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+my name is noyan
```
Comment (BackAged) [inline noyan.txt:1]: @noyon57 As @Neaz-Mahmood has already said, first letter of name should be capital letter, would you mind fix that?

**Review by Neaz-Mahmood — APPROVED** [review]
Looks good.

File: noyan.txt (hunk 1 above)
Comment (noyon57) [reply noyan.txt]: done!

**Review by Roman-Shahrear — APPROVED** [review]

**Review by BackAged — APPROVED** [review]
Awesome

**Pull request merged**